### ISSUE TITLE
ustream.tv breaks video

### ISSUE THREAD

**Issue by terrorist96** [EFForg/privacybadger#796]
https://model3.tesla.com

need to input info to see video.

`www.ustream.tv` needs to be green
the following need to be yellow:
`rgw.ustream.tv`
`tcdn.ustream.tv`
`vod11.ustream.tv`

Side issue: I'm getting a notice from Chrome again saying "Warning:
This extension is slowing down Google Chrome. You should disable it to restore Google Chrome's performance."

**Comment by cooperq** [comment]
I don't see ustream at all on that site, can you provide another example.

**Comment by terrorist96** [comment]
Well at the time of this report, you had to input basic info and you could see the video of the model 3 unveiling via a ustream embedding. It looks like they removed that and it's solely a page to reserve the model 3 now. I don't have another example to give you right now.

**Comment by terrorist96** [comment]
@cooperq Here's another example:
http://plasticmind.com/labs/ustream/

**Comment by terrorist96** [comment]
Another example:
http://www.iflscience.com/space/watch-live-as-a-spacecraft-leaves-the-iss-today-with-a-fire-experiment-on-board/

**Comment by cooperq** [comment]
are you certain that www.ustream.tv needs to be green? If that's the case then we need to see if we can add ustream to the social widget replacement list.

**Comment by terrorist96** [comment]
Yes, it needs to be green. You can test for yourself on Chrome (which has flash built in, so you don't need to install it)

**Comment by terrorist96** [comment]
Another example: http://streamfare.com/univisioni.html

`cdngw.ustream.tv`
`static-cdn1.ustream.tv`
`uhs-akamai.ustream.tv`
`www.ustream.tv`

**Comment by terrorist96** [comment]
http://streamfare.com/telemundo.html

**Comment by terrorist96** [comment]
@ghostwords This issue is over 2 years old now. Can you please fix?

Still reproducible with these URLs:
http://plasticmind.com/labs/ustream/
http://www.iflscience.com/space/watch-live-as-a-spacecraft-leaves-the-iss-today-with-a-fire-experiment-on-board/

**Comment by terrorist96** [comment]
```
**** ACTION_MAP for ustream
VM1584:5 cdngw.ustream.tv {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1519313403193,
  "userAction": ""
}
VM1584:5 ic.a2f58271.0b4053.1.ustreamuhs.loris.llnwd.net {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1493078525250,
  "userAction": ""
}
VM1584:5 static-cdn1.ustream.tv {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1520226520197,
  "userAction": ""
}
VM1584:5 uhs-akamai.ustream.tv {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1519530829250,
  "userAction": ""
}
VM1584:5 ustream.tv {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1492703230911,
  "userAction": ""
}
VM1584:5 www.ustream.tv {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1523093254540,
  "userAction": ""
}
VM1584:5 cdn1.ustream.tv {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1523194381365
}
VM1584:5 static-cdn2.ustream.tv {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1523540662675
}
VM1584:7 **** SNITCH_MAP for ustream
VM1584:9 ustream.tv [
  "iflscience.com",
  "kcaaradio.com",
  "tesla.com",
  "plasticmind.com"
]
```

**Comment by ghostwords** [comment]
@terrorist96 Do you have any Ustream/IBM Cloud Video examples that won't disappear quickly (and aren't Flash-only)?

Since a domain needs to be green, widgets (#1467) are the only existing way to handle this situation.
